network/alsp/manager/manager_helper_test.go

```diff
@@ -0,0 +1,126 @@
+package alspmgr_test
```

Comment:
We can probably move these fixtures back to the manager_test.go file or rename this file to something that indicates that this is a test fixture file.

Reply:
Moved them back f7b7b62c4c696eab9f87a66fd25a77df71f8cdad